application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +48,12 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
```

Comment:
u dont have to specify all those stuff just do 
```java
queue(any -> {}, err -> { ... })
```

Reply:
done updated